web/docs/tql2/operators/save_azure_blob_storage.md

```diff
@@ -0,0 +1,55 @@
+# save_azure_blob_storage
+
+Saves bytes to an Azure Blob Storage.
```

Comment:
Same thing here as before. There might be multiple stores, but Azure Blob Storage is the service, of which there is only one. So not "an".